tests/network-tests/src/flows/clis/channelsAndVideos.ts

```diff
@@ -0,0 +1,278 @@
+import { FlowProps } from '../../Flow'
+import { extendDebug } from '../../Debugger'
+import { BuyMembershipHappyCaseFixture } from '../../fixtures/membership/BuyMembershipHappyCaseFixture'
+import { FixtureRunner } from '../../Fixture'
+import { assert } from 'chai'
+import { Utils } from '../../utils'
+import { statSync, readFileSync } from 'fs'
+import BN from 'bn.js'
+import { createJoystreamCli } from '../utils'
+import { createType } from '@joystream/types'
+import { u8aConcat, u8aFixLength } from '@polkadot/util'
+import { ArgusApi } from '../../ArgusApi'
+import {
+  ChannelCreationInputParameters,
+  ChannelUpdateInputParameters,
+  VideoInputParameters,
+} from '@joystream/cli/src/Types'
+
+export default async function createAndUpdateChannel({ api, query, env }: FlowProps): Promise<void> {
+  const debug = extendDebug('flow:createChannel')
```

Comment:
Shouldn't this also be renamed to:
```suggestion
  const debug = extendDebug('flow:channelAndVideos')
```